static/js/settings_org.js

```diff
@@ -103,6 +103,40 @@ exports.populate_auth_methods = function (auth_methods) {
     }
 };
 
+var property_types = {
+            add_emoji_by_admins_only: {type: 'bool',
+                                checked_msg: "Only administrators may now add new emoji!",
+                                unchecked_msg: "Any user may now add new emoji!"},
```

Comment:
these messages should be tagged for translation using `i18n.t()`.  I think that will require you to move this to use `ensure_i18n`.